app/assets/stylesheets/application.css

```diff
@@ -4,12 +4,6 @@
  *
  * Any CSS (and SCSS, if configured) file within this directory, lib/assets/stylesheets, or any plugin's
  * vendor/assets/stylesheets directory can be referenced here using a relative path.
- *
- * You're free to add application-wide styles to this file and they'll appear at the bottom of the
- * compiled file so the styles you add here take precedence over styles defined in any other CSS
- * files in this directory. Styles in this file should be added after the last require_* statement.
- * It is generally better to create a new file per style scope.
- *
  *= require_tree .
  *= require_self
  */
```

Comment:
- [ ] On the "Add a new category" page, Please make sure to add a "Back" button (<) and clicking on it the user should navigate to the home page as requested in the project requirements. Currently, I can't see the back button